examples/src/components/GeoPushpinChartConfigurationPointsExample.jsx

```diff
@@ -0,0 +1,113 @@
+import { MAPBOX_TOKEN, cityCoordinatesUri, populationUri, cityNamesUri } from "../utils/fixturesGeoChart";
+
+const sizeMeasure = Model.measure(populationUri)
+    .format("#,##0.00")
+    .aggregation("sum")
+    .alias("Population");
+const locationAttribute = Model.attribute(cityCoordinatesUri).localIdentifier("location");
+const tooltipTextAttribute = Model.attribute(cityNamesUri).localIdentifier("tooltipText");
+
+export class GeoPushpinChartConfigurationPointsExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            config: {
+                tooltipText: tooltipTextAttribute,
+                mapboxToken: MAPBOX_TOKEN,
+                points: {
+                    groupNearbyPoints: true,
+                },
+            },
+            customGroupNearbyPointsUsed: false,
+        };
+    }
+    onLoadingChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onLoadingChanged", ...params);
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onError", ...params);
+    }
+
+    onZoomChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onZoomChanged", ...params);
+    }
+
+    onCenterPositionChanged(...params) {
+        // eslint-disable-next-line no-console
+        return console.log("GeoPushpinChartConfigurationPointsExample onCenterPositionChanged", ...params);
+    }
+
+    onCustomPointsSizeTabSelect = () => {
+        const { config: currentConfig } = this.state;
+        this.setState({
+            config: {
+                ...currentConfig,
+                points: {
+                    minSize: "0.5x",
+                    maxSize: "1.5x",
+                },
+            },
+            customGroupNearbyPointsUsed: false,
+        });
+    };
+
+    onCustomGroupNearbyPointsSelect = () => {
+        const { config: currentConfig } = this.state;
+        this.setState({
+            config: {
+                ...currentConfig,
+                points: {
+                    groupNearbyPoints: false,
+                },
+            },
+            customGroupNearbyPointsUsed: true,
+        });
+    };
+
+    render() {
+        const style = { height: "500px" };
+        const { config: geoConfig, customGroupNearbyPointsUsed } = this.state;
+        // eslint-disable-next-line no-console
+        return (
+            <div className="s-geo-chart">
+                <button className="s-change-points-size" onClick={this.onCustomPointsSizeTabSelect}>
+                    Custom Points Size
+                </button>
+
+                <button
+                    className="s-change-group-neaerby-points"
+                    onClick={this.onCustomGroupNearbyPointsSelect}
```

Comment:
how about the name `this.toggleGroupNearbyPoints`?

Reply:
updated